### PR TITLE
Migrate ant build system to gradle

### PR DESCRIPTION
**Pull request by desyncr**
Work in progress: There are a few things that need to be migrated such as:

			<sysproperty key="benchmark" value="${benchmark}" />
			<sysproperty key="extensiveTesting" value="${extensiveTesting}" />

These values are passed to the junit tests.

This PR will mess up all pending PRs.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+compileJava {
+    sourceCompatibility = 1.7
+    targetCompatibility = 1.7
+}
+
+version = "37"
+
+repositories {
+    mavenLocal()
```
Comment (Bombe) [inline build.gradle:20]: This shouldn’t be included here.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    manifest {
+        attributes 'Plugin-Main-Class': 'plugins.Library.Main',
+                   'Main-Class': 'plugins.Library.ui.TestInterface',
+                   'Required-Node-Version': '1239',
+                   'Implementation-Version': version,
+                   'Built-By': System.getProperty('user.name'),
+                   'Built-Date': getMTime("src/main/java/plugins/Library/Version.java"),
+                   'Built-JDK': System.getProperty('java.version')
+    }
+    from('src') {
```
Comment (Bombe) [inline build.gradle:75]: Everything from the resources directory is included automatically.

**Review by Bombe — CHANGES_REQUESTED** [review]
I can’t comment on files that have been moved without changes so I’ll mention it here: everything that is *not* a .java file belongs in `src/*/resources`, *not* in `src/*/java`!

File: build.gradle (hunk 1 above)
Comment (desyncr) [reply build.gradle]: Done :+1:

File: build.gradle (hunk 2 above)
Comment (desyncr) [reply build.gradle]: True. Done! :+1:

**Comment by desyncr** [comment]
@Bombe Thanks for the heads up. Moved everything under the corresponding directory. Also added gradle witness verification.

**Pull request merged**